sql/moz-fx-data-shared-prod/braze_external/users_previous_day_snapshot_v2/schema.yaml

```diff
@@ -36,5 +36,5 @@ fields:
   name: fxa_lang
   type: STRING
 - mode: NULLABLE
-  name: first_service
+  name: fxa_first_service
```

Comment:
Is this schema already deployed? We may need to drop the table or just the column and redeploy the schema for this schema change to work.

Reply:
yes, it's already deployed. it drops the snapshot table before it creates it each time (b/c you can't overwrite one)...should create fine this time :)